contrib/examples/apiserver/broker.yaml

```diff
@@ -8,6 +8,6 @@ spec:
   # service-catalog will use the contents of the secret. The secret should have "username"
   # and "password" keys
   authInfo:
-    basicAuthSecret:
+    authSecret:
```

Comment:
`authInfo.authSecret` looks redundant.
Shall I rename it to just `secret` or `secretRef`?